avatars/avatars.js

```diff
@@ -993,7 +993,7 @@ class Avatar {
     this.lastNeedsHeadTarget = false;
     this.lastHeadTargetTime = -Infinity;
 
-    this.manuallySetMouth=false;
+    this.disableAudioWorkerSetVolume=false;
```

Comment:
It is not good practice to have double negatives in variable names. The variable name should be the positive version, `enable`.